src/Nav/Navbar.scss

```diff
@@ -0,0 +1,8 @@
+.NavbarLogo {
+  height: 5vh;
```

Comment:
This means that as the browser gets taller the logo gets taller, I don’t think we want that.